general/config-create.yaml

```diff
@@ -12,5 +12,5 @@ install:
 #  tty: ttyS1,115200n8   # For machines without a VGA console
 
   vip: 10.10.0.19
-  vip_mode: DHCP                   # Or static
```

Comment:
The problem was not here. Rather, it was the `method` attribute under `networks` when enabling additional NICs here  https://github.com/harvester/ipxe-examples/blob/main/vagrant-pxe-harvester/ansible/roles/harvester/templates/config-create.yaml.j2#L16 and https://github.com/harvester/ipxe-examples/blob/main/vagrant-pxe-harvester/ansible/roles/harvester/templates/config-join.yaml.j2#L17. I think there's a bug in the installer code. It's supposed to do case-insensitive comparison.

Reply:
Updated.